frontend/src/FirstRun/AuthenticationRequiredModalContentConnector.tsx

```diff
@@ -0,0 +1,43 @@
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { clearPendingChanges } from 'Store/Actions/baseActions';
+import {
+  fetchGeneralSettings,
+  saveGeneralSettings,
+  setGeneralSettingsValue,
+} from 'Store/Actions/settingsActions';
+import { fetchStatus } from 'Store/Actions/systemActions';
+import createSettingsSectionSelector from 'Store/Selectors/createSettingsSectionSelector';
+import AuthenticationRequiredModalContent from './AuthenticationRequiredModalContent';
+
+const SECTION = 'general';
+
+const selector = createSettingsSectionSelector(SECTION);
+
+export default function AuthenticationRequiredModalContentConnector() {
+  const props = useSelector(selector);
+  const dispatch = useDispatch();
+
+  useEffect(() => {
+    dispatch(fetchGeneralSettings());
+  }, [dispatch]);
+
+  useEffect(() => {
+    return () => {
+      dispatch(clearPendingChanges());
+    };
+  }, [dispatch]);
+
+  return (
+    <AuthenticationRequiredModalContent
+      dispatchFetchStatus={() => dispatch(fetchStatus())}
+      onInputChange={(args: any) => {
+        dispatch((setGeneralSettingsValue as any)(args));
```

Comment:
It seems that this dispatcher wasn't called before  
It now triggers a call to the redux devtools on each value change

The typing given by `react-actions` doesn't match the doc usage